tests/console/gpg.pm

```diff
@@ -155,6 +156,19 @@ sub run {
     # Obtain GnuPG version
     my $gpg_version_output = script_output("gpg --version");
     my ($gpg_version) = $gpg_version_output =~ /gpg \(GnuPG\) (\d\.\d)/;
+    record_info('gpg version', "Version of Current gpg package: $gpg_version");
```

Comment:
According to the verify run https://openqa.opensuse.org/tests/2120886#step/gpg/2 , the gpg version is '2.3.4', but in the record message it is '2.3'. There is something wrong in the above regex. Please check that.

Reply:
Great comment, I think I miss 1 digit check, let me fix it, thanks @Amrysliu

Reply:
All VR is finished, resolved.